src/mixxx.cpp

```diff
@@ -366,13 +366,13 @@ void MixxxMainWindow::initialize(QApplication* pApp, const CmdlineArgs& args) {
 
     // Load skin to a QWidget that we set as the central widget. Assignment
     // intentional in next line.
-    if (!(m_pWidgetParent = m_pSkinLoader->loadDefaultSkin(this, m_pKeyboard,
-                                                           m_pPlayerManager,
-                                                           m_pControllerManager,
-                                                           m_pLibrary,
-                                                           m_pVCManager,
-                                                           m_pEffectsManager,
-                                                           m_pRecordingManager))) {
+    if (!(m_pWidgetParent = m_pSkinLoader->loadConfiguredSkin(this, m_pKeyboard,
+                                                              m_pPlayerManager,
```

Comment:
Thats why the parenthese alignment ... to much noise ;-)

Reply:
I see what you mean here. I don't think we have a specific rule in the style guide about how to indent function calls with many parameters. How should it be written?

Reply:
You can keep it like it is. It was only a kidding comment. 

This is the way Eclipse auto fomats it:
```C++
QWidget* SkinLoader::loadConfiguredSkin(QWidget* pParent,
        KeyboardEventFilter* pKeyboard, PlayerManager* pPlayerManager,
        ControllerManager* pControllerManager, Library* pLibrary,
        VinylControlManager* pVCMan, EffectsManager* pEffectsManager,
        RecordingManager* pRecordingManager) {
```

This is my favourite:
```C++
QWidget* SkinLoader::loadConfiguredSkin(
        QWidget* pParent,
        KeyboardEventFilter* pKeyboard, 
        PlayerManager* pPlayerManager,
        ControllerManager* pControllerManager, 
        Library* pLibrary,
        VinylControlManager* pVCMan, 
        EffectsManager* pEffectsManager,
        RecordingManager* pRecordingManager) {
```